src/stores/widgets/StopGapWidget.ts

```diff
@@ -255,8 +255,12 @@ export class StopGapWidget extends EventEmitter {
             });
         }
     };
-
-    public start(iframe: HTMLIFrameElement) {
+    /**
+     * This starts the messaging for the widget if it is not in the state `started` yet.
+     * @param iframe the iframe the widget should use
+     * @returns
```

Comment:
Hmm, maybe leave out returns here as well...? It does return sometimes, but not sure what to say about it...

```suggestion
```